frame/dapps-staking/src/pallet/mod.rs

```diff
@@ -156,6 +156,21 @@ pub mod pallet {
     pub(crate) type ContractLastStaked<T: Config> =
         StorageMap<_, Twox64Concat, SmartContract<T::AccountId>, EraIndex>;
 
+    #[pallet::type_value]
+    pub(crate) fn PreApprovalOnEmpty() -> bool {
+        false
+    }
```

Comment:
I think you can omit this - default bool value is already false.

Reply:
I know, but I think we will set it to true when we launch the network. This is a placeholder to be ready for that change. UTs were also adapted to support both cases